sdk/python/kfp/_runners.py

```diff
@@ -49,6 +50,36 @@ def run_pipeline_func_on_cluster(
       pipeline_conf: Optional. kfp.dsl.PipelineConf instance. Can specify op
         transforms, image pull secrets and other pipeline-level configuration
         options.
-    '''
+    """
     kfp_client = kfp_client or Client()
-    return kfp_client.create_run_from_pipeline_func(pipeline_func, arguments, run_name, experiment_name, pipeline_conf)
+    return kfp_client.create_run_from_pipeline_func(
+        pipeline_func, arguments, run_name, experiment_name, pipeline_conf
+    )
+
+
+def run_pipeline_func_locally(
+    pipeline_func: Callable,
+    arguments: Mapping[str, str],
+    local_client: LocalClient = None,
+    local_env_images: List[str] = None,
+):
+    """Run kubeflow pipeline in docker or locally
+
+    Feature stage:
+    [Alpha](https://github.com/kubeflow/pipelines/blob/master/docs/release/feature-stages.md#alpha)
```

Comment:
If there are any limitations of local runner, it would be good to document that here as well. E.g., certain components/ops don't work when run locally?